lib/cmds/exec_strategy.js

```diff
@@ -64,35 +64,41 @@ module.exports = async (ds, ws, msg) => {
     symbol, tf, rangeString(start, end)
   )
 
-  const candleData = await Candle.getInRange([
-    ['exchange', '=', exchange],
-    ['symbol', '=', symbol],
-    ['tf', '=', tf]
-  ], {
-    key: 'mts',
-    start,
-    end
-  }, {
-    orderBy: 'mts',
-    orderDirection: 'asc'
-  })
-
-  debug('loaded %d candles', candleData.length)
-
-  const tradeData = await Trade.getInRange([
-    ['exchange', '=', exchange],
-    ['symbol', '=', symbol]
-  ], {
-    key: 'mts',
-    start,
-    end
-  }, {
-    orderBy: 'mts',
-    orderDirection: 'asc'
-  })
+  let candleData = []
+  if (includeCandles) {
+    candleData = await Candle.getInRange([
+      ['exchange', '=', exchange],
+      ['symbol', '=', symbol],
+      ['tf', '=', tf]
+    ], {
+      key: 'mts',
+      start,
+      end
+    }, {
+      orderBy: 'mts',
+      orderDirection: 'asc'
+    })
+
+    debug('loaded %d candles', candleData.length)
+  }
 
-  tradeData.sort((a, b) => a.mts - b.mts)
-  debug('loaded %d trades', tradeData.length)
+  let tradeData = []
+  if (includeTrades) {
+    tradeData = await Trade.getInRange([
+      ['exchange', '=', exchange],
+      ['symbol', '=', symbol]
+    ], {
+      key: 'mts',
+      start,
+      end
+    }, {
+      orderBy: 'mts',
+      orderDirection: 'asc'
+    })
+
+    tradeData.sort((a, b) => a.mts - b.mts)
```

Comment:
Just trying to wrap my head around this. You already orderBy mts asc. Is this sort needed here? I might be just missing something obvious here.

Reply:
good call, this was taken from the code i ported from the frontend some time ago, without it the chart drawing would mess up

Reply:
i glanced into it back then and afaik it reverses the order of the data

Reply:
i double checked it, its superfluous, removed!